autointent/configs/_transformers.py

```diff
@@ -0,0 +1,107 @@
+from enum import Enum
+from typing import Any
+
+from pydantic import (
+    BaseModel,
+    Field,
+    PositiveInt,
+)
+from typing_extensions import Self
+
+
+class ModelConfig(BaseModel):
+    batch_size: PositiveInt = Field(32, description="Batch size for model inference.")
+    max_length: PositiveInt | None = Field(None, description="Maximum length of input sequences.")
+
+
+class STModelConfig(ModelConfig):
```

Comment:
Тк EmbedderConfig и Crossencoder config имеет, то можно и сюда добавить, а они переопределят
```suggestion
class STModelConfig(ModelConfig):
    model_name: str
```